app/session.py

```diff
@@ -0,0 +1,90 @@
+import uuid
```

Comment:
I wonder if this is is based on an existing implementation or written from scratch?

Reply:
`uuid` is a standard python module... Not sure I follow..

Reply:
ah this comment is rather for the whole file, I just don't know where to put it :)